TheJumpCloud/support
# Import/Update CSV attribute bug fix

## Issues
* [CUT-4320](https://jumpcloud.atlassian.net/browse/CUT-4320) - CUT-4320-CustomAttribute-Bug-Import-Update-User-CSV

## What does this solve?
Fixes an issue when some Attribute fields are not null or empty which results to user not being updated
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Import new users with multiple attributes where atleast one attribute name and value is empty. Check CSV below for example
[NewUserCSV.csv](https://github.com/user-attachments/files/17385370/NewUserCSV.csv)
2. Validate that all users are created even with some Attributes that are empty
3. Run Update-JCUsersFromCSV where one of the Attribute fields on your CSV to be empty. CSV below for example
[UpdateUser.csv](https://github.com/user-attachments/files/17385411/UpdateUser.csv)
4. Validate the users custom attributes are updated
## Screenshots
Attribute 2 is empty on this Import. The new bug fix reorganized the row to keep the numerical formality which renamed Attribute3 to Attribute2,
![image](https://github.com/user-attachments/assets/d200fa78-c121-4f31-bfaf-cf5ce97b6350)


[CUT-4320]: https://jumpcloud.atlassian.net/browse/CUT-4320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
